### PR TITLE
:bug: FIX: tsconfig not written in init cmd due to comment in new kit…

### PR DESCRIPTION
**Pull request by jycouet**
… version

Fixes #TICKET

### To help everyone out, please make sure your PR does the following:

- [ ] Update the first line to point to the ticket that this PR fixes
- [ ] Add a message that clearly describes the fix
- [ ] If applicable, add a test that would fail without this fix
- [ ] Make sure the unit and integration tests pass locally with `pnpm run tests` and `cd integration && pnpm run tests`
- [ ] Includes a changeset if your fix affects the user with `pnpm changeset`

### CODE REVIEW

**Comment by AlecAivazis** [comment]
I'm starting to think we're being too smart in init. What do you think if we just always write the new file instead of trying to figure out if it's "safe". If we destroy users code they set up, they can always look at the diff to get it back... Thoughts?

File: packages/houdini/src/cmd/init.ts [hunk 1]
```diff
@@ -291,8 +291,13 @@ async function tjsConfig(targetPath: string, framework: 'kit' | 'svelte') {
 
 	// check if the tsconfig.json file exists
 	try {
-		const tjsConfigFile = await fs.readFile(configFile)
+		let tjsConfigFile = await fs.readFile(configFile)
 		if (tjsConfigFile) {
+			// remove all comments to be able to parse the file, and add stuff to it.
+			tjsConfigFile = tjsConfigFile.replace(
+				/\\"|"(?:\\"|[^"])*"|(\/\/.*|\/\*[\s\S]*?\*\/)/g,
```
Comment (AlecAivazis) [inline packages/houdini/src/cmd/init.ts:298]: I feel like we should have a test that verifies this regex

**Comment by jycouet** [comment]
I'm totally for it. 

1/ Check if nothing is pending in GIT
2/ Questions
3/ Write files

Mermaid 👇

```mermaid
flowchart LR

A[init] -->B(GIT Status)
B --> |not clean| A
B --> C{is Running}
C --> |no| D{schemaPath}
D --> |not correct| D
D --> F(done)
C --> |yes| E{url}
E --> |not correct| E
E --> F(DONE)
```

---

I think that this PR could be merged to fix today's issue.

And tackle the "bigger view" in a separate PR with also plugin ideas?

**Review by AlecAivazis — APPROVED** [review]
Yea that sounds good! Thanks for adding the test 👍

**Comment by AlecAivazis** [comment]
Yea I like the additional branch for a local schema path

**Pull request merged**